calico/_includes/charts/tigera-operator/README.md

```diff
@@ -114,6 +114,10 @@ certs:
     commonName:
     caBundle:
 
+# We usually recommend not to specify default resources and to leave this as a conscious
```

Comment:
"Resources for the tigera/operator pod itself. By default, no resource requests or limits are specified."

I'd suggest saying something like this ^

Reply:
Yes, that sounds better, instead of the default comment from helm.